orchestrate-cockroachdb-with-kubernetes.md

```diff
@@ -40,7 +40,7 @@ Feature | Description
 --------|------------
 instance | A physical or virtual machine. In this tutorial, you'll run a Kubernetes script from your local workstation that will create 4 GCE or AWS instances and join them into a single Kubernetes cluster. 
 [pod](http://kubernetes.io/docs/user-guide/pods/) | A pod is a group of one of more Docker containers. In this tutorial, each pod will run on a separate instance and contain one Docker container running a single CockroachDB node. You'll start with 3 pods and grow to 4.
-[PetSet](http://kubernetes.io/docs/user-guide/petset/) | A PetSet is a group of pods treated as stateful units, where each pod has distinguishable network identity and always binds back to the same persistent storage on restart. PetSets are an alpha feature as of Kubernetes version 1.4. As such, they may not be available in some production-ready hosting environments.
+[StatefulSet](http://kubernetes.io/docs/concepts/abstractions/controllers/statefulsets/) | A StatefulSet is a group of pods treated as stateful units, where each pod has distinguishable network identity and always binds back to the same persistent storage on restart. StatefulSets are an beta feature as of Kubernetes version 1.5. As such, they may not be available in some production-ready hosting environments.
```

Comment:
s/an beta/a beta/

And remove the part about not being available. Beta features are available in all kubernetes clusters.